project/SotaBuild.scala

```diff
@@ -89,6 +91,18 @@ object SotaBuild extends Build {
       testOptions in RandomTests += Tests.Argument(TestFrameworks.ScalaTest, "-n", "RandomTest"),
       parallelExecution in Test := true,
       dockerExposedPorts := Seq(8081),
+      dockerCommands := dockerCommands.value.filterNot {
```

Comment:
Then we could just use `dockerEntrypoint` instead of filtering out and adding again.